packages/upload/src/vaadin-upload.js

```diff
@@ -902,6 +898,26 @@ class Upload extends ElementMixin(ThemableMixin(ControllerMixin(PolymerElement))
     }
   }
 
+  /** @private */
+  _getAcceptRegex() {
+    if (!this.accept) {
+      return;
+    }
```

Comment:
I wonder if this part is necessary. The original version didn't have it.

Reply:
An empty string is produced when an empty `this.accept` if split. There isn't really anything to process though. Therefore, I thought that it would be cleaner to just return without getting to the processing part.

Reply:
Makes sense. Let's keep it.